### DISCUSSION TITLE
虚

### DISCUSSION THREAD

**Discussion by mayehaipa** [share121/inter-knot#5139] [其他]
家人们谁懂啊，一天两次导致我身体走个楼梯都很累，不知道有没有什么恢复的方法

**Comment by mayehaipa** [comment]
111加急兄弟们，顺便问问怎么改头像

**Comment by qingshuiglaze** [comment]
节欲，然后去锻炼吧，这是最好的办法，吃药也没这个效果好<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >改头像在github上，点自己的头像往下找找就行

**Comment by faatong** [comment]
绳网有搜索功能，搜头像，里面有图文教程<img src="https://img-static.mihoyo.com/communityweb/upload/392df119e6bb2c758b5d8bf25e8d58a1.png" alt="盯" width="75" >
